rotkehlchen/db/upgrades/v34_v35.py

```diff
@@ -384,11 +385,17 @@ def _add_defillama_to_all_oracles(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_defillama_to_all_oracles')
 
 
-def _reset_decoded_events(write_cursor: 'DBCursor') -> None:
+def _reset_decoded_events(db: 'DBHandler', write_cursor: 'DBCursor') -> None:
     """Reset all non-user customized decoded events"""
     log.debug('Enter _reset_decoded_events')
+    tx_hashes = []
+    with db.conn.read_ctx() as cursor:
+        for entry in cursor:
```

Comment:
I don't understand this here. What does cursor have? You haven't executed any query right?

Reply:
wtf? :sweat: Where did my query go? I did not commit it!?